voxel51/users/api.py

```diff
@@ -459,24 +459,36 @@ def delete_data(self, data_id):
         _validate_response(res)
 
     def batch_update_data_ttl(self, data_ids, days):
-        '''Updates the expiration date of all specified data.
+        '''Updates the expiration date of the data with the given IDs.
+
+        To decrease the lifespan of the data, provide a negative number. Note
+        that if the expiration date of data after modification is in the past,
+        the data will be deleted.
 
         Args:
             data_ids (list): the data IDs
             days (float): the number of days by which to extend the lifespan
                 of the data
 
+        Returns:
+            a list of True/False indicating which data IDs were successfully
```

Comment:
It's actually a dictionary of `id` -> (some status object) - we could reformat it to be more Python-friendly (I'll pull up an example when I get a chance)